src/Raven.Server/RavenServer.cs

```diff
@@ -1593,6 +1593,18 @@ internal AuthenticateConnection AuthenticateConnectionCertificate(X509Certificat
             {
                 authenticationStatus.Status = AuthenticationStatus.ClusterAdmin;
             }
+            else if (CertificateHasWellKnownIssuer(certificate, out var issuer))
+            {
+                if (_authAuditLog.IsInfoEnabled)
+                {
+                    _authAuditLog.Info(
+                        $"Connection from {GetRemoteAddress(connectionInfo)} with new certificate '{certificate.Subject} ({certificate.Thumbprint})' which is not registered in the cluster. " +
+                        "Allowing the connection based on the certificate's *issuer* which is trusted by the cluster. " +
+                        $"Registering the new certificate explicitly based on permissions of existing certificate '{issuer}'. Security Clearance: {AuthenticationStatus.ClusterAdmin}");
```

Comment:
That issuer here is only a subject. Can we also include its thumbprint?

Reply:
Yes